source/educators/references/course_development/develop_course_sections.rst

```diff
@@ -106,3 +104,25 @@ A section can contain a unit that is hidden from learners and available to
 members of the course team only. That unit is not visible to learners,
 regardless of the release date of the section or subsection.
 
+
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```

Comment:
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
 :ref:`Course Outline` (concept)
 
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams <Resources for edx org Course Teams>` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

Reply:
Resolved spacing issues in See Also section.